main.nf

```diff
@@ -775,7 +776,8 @@ process ConcatVCF {
 	done
 	cat header raw_calls > unsorted.vcf
 	java -jar \${PICARD_HOME}/picard.jar SortVcf I=unsorted.vcf O=$outputFile
-    rm unsorted.vcf
+	rm unsorted.vcf
+	gzip -v $outputFile
```

Comment:
Can you pipe into gzip to avoid some I/O? I think picard accepts `O=/dev/stdout`